### DISCUSSION TITLE
An inquiry about generate a quasi-3D bicrystal RVE in MOOSSE

### DISCUSSION THREAD

**Discussion by NINGYURICHARD** [idaholab/moose#26821] [Q&A Modules: Solid mechanics]
Hi,

I am trying to generate a quasi-3D bicrystal RVE (i.e. one element in the transverse direction) in MOOSE. The attached figure 
![bicrystal](https://github.com/idaholab/moose/assets/74522792/c8253a81-cf55-4403-9dad-96726dfc2973)
 is the example I want to generate for compression or tension simulation. I am a beginner of MOOSE, could you please tell me whether MOOSE can generate such bicrystal RVE? Thank you very much!

Best Regards,
Richard

**Comment by GiudGiud** [comment]
Hello

You can make the 2D system in moose either with a fitted circular mesh or from a cartesian regular mesh and a bicrystal initial condition. 

There should be a tutorial in the phase field module to do this
https://mooseframework.inl.gov/modules/phase_field/index.html

Guillaume
